libs/docs/core/date-picker/date-picker-docs.component.html

```diff
@@ -17,6 +17,15 @@
 
 <separator></separator>
 
+<fd-docs-section-title id="multi" componentName="date-picker"> Multi Date Picker </fd-docs-section-title>
+<description> Allows the selection of a multiple dates, just pass <b>allowMultipleSelection</b> as true. </description>
```

Comment:
Can you make this a bit more verbose (explain what the feature is)? Or link it to the feature of the Calendar